kpops/utils/gen_schema.py

```diff
@@ -138,9 +138,7 @@ def gen_pipeline_schema(
         )
         core_schema: DefinitionsSchema = component.__pydantic_core_schema__  # pyright:ignore[reportGeneralTypeIssues]
 
-        model_schema: ModelFieldsSchema = core_schema["schema"]["schema"]["schema"][  # pyright:ignore[reportGeneralTypeIssues,reportTypedDictNotRequiredAccess]
-            "schema"
-        ]
```

Comment:
Why did this change again?

Reply:
I believe it's because the pydantic model validator is not used anymore. Apparently it causes the schema to get nested like this